app/javascript/src/views/favorites.vue

```diff
@@ -0,0 +1,204 @@
+<template>
```

Comment:
Definition for rule 'vue/max-len' was not found  vue/max-len

Reply:
Se supone que esto estaba arreglado 😢 Pero el monito piensa otra cosa